prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/DefaultFilterTests.kt

```diff
@@ -170,7 +171,28 @@ class DefaultFilterTests {
         val result = engine.evaluateFilterCondition(emptyList(), bundle, false)
 
         // assert
-        assert(!result)
+        assertThat(result).isFalse()
+    }
+
+    @Test
+    fun `test evaluateFilterCondition reverse filter`() {
+        // set up
+        val settings = FileSettings().loadOrganizations(oneOrganization)
+        val one = Schema(name = "None", topic = Topic.FULL_ELR, elements = emptyList())
+        val metadata = Metadata(schema = one)
+
+        val engine = spyk(makeFhirEngine(metadata, settings, TaskAction.route) as FHIRRouter)
+        val bundle = FhirTranscoder.decode(fhirCodeP)
+
+        // act
+        val useDefaultResult = engine.evaluateFilterCondition(emptyList(), bundle, false, true)
+        // assert
+        assertThat(useDefaultResult).isTrue()
+
+        // act
+        val useBundleResult = engine.evaluateFilterCondition(engine.processingModeFilterDefault, bundle, false, false)
```

Comment:
Not asking you to change this right now, but there is no need for the useBundleResult val.  Just put the code inside the assertThat().  Same for all the other below.  I know you did not write those.